llvm/include/llvm/ADT/StableHashing.h

```diff
@@ -53,12 +53,6 @@ inline stable_hash stable_hash_combine(stable_hash A, stable_hash B,
 // Removes suffixes introduced by LLVM from the name to enhance stability and
 // maintain closeness to the original name across different builds.
 inline StringRef get_stable_name(StringRef Name) {
-  // Return the part after ".content." that represents contents.
-  auto [P0, S0] = Name.rsplit(".content.");
-  if (!S0.empty())
-    return S0;
-
```

Comment:
Just a drive-by comment: It would be nice to have changes to core llvm headers like this in a separate PR, so that reverting doesn't cause everyone to rebuild a large portion of the build tree.

Reply:
Sure. I will follow this up!